lib/tasks/cucumber.rake

```diff
@@ -1,65 +1,15 @@
-# IMPORTANT: This file is generated by cucumber-rails - edit at your own peril.
-# It is recommended to regenerate this file in the future when you upgrade to a
-# newer version of cucumber-rails. Consider adding your own code to a new file
-# instead of editing this one. Cucumber will automatically load all features/**/*.rb
-# files.
+unless Rails.env.production?
```

Comment:
It might be helpful to link to https://github.com/alphagov/collections/pull/1683 and https://github.com/alphagov/collections/commit/6a13dbf5234df0745717e742222b78d229346607 to show there is prior art for this.

Reply:
Sure, I've copied the commit message so it's explained in isolation.